sdk/dotnet/Pulumi/Deployment/Deployment_Call.cs

```diff
@@ -53,8 +53,10 @@ private Output<T> Call<T>(string token, CallArgs args, Resource? self, CallOptio
             }
 
             var (serialized, argDependencies) = await SerializeFilteredPropertiesAsync(
-    				$"call:{token}",
-    				argsDict, _ => true, keepResources: true).ConfigureAwait(false);
+                    $"call:{token}",
+                    argsDict, _ => true,
+                    keepResources: true,
+                    keepOutputValues: true).ConfigureAwait(false);
```

Comment:
So Call assumes the monitor supports output values? If memory serves me correctly this might be reasonable but good to double-check.

Reply:
That is my understanding. It is how it works in the other SDKs.